hbase_sql/src/main/java/com/splicemachine/compactions/SpliceDefaultCompactionPolicy.java

```diff
@@ -48,7 +48,9 @@ public CompactionRequestImpl selectCompaction(Collection<HStoreFile> candidateFi
         HRegion region = store.getHRegion();
         String storeName = store.getColumnFamilyName();
         scr.setIsMajor(cr.isMajor(), cr.isAllFiles());
-        scr.setOffPeak(cr.isOffPeak());
+        // Ignoring cr.isOffPeak() and hardcoding true because SpliceCompactionRequest.setOffPeak is hijacked to run
+        // SpliceCompactionRequest.afterExecute if isOffPeak(false) is called.
+        scr.setOffPeak(true);
```

Comment:
Are there any interactions with real off-peak compactions scheduled from HBase?

Reply:
Based on what I could see, this parameter was ignored anyway, so it should not have any impact.